platform/resources/RetirementPipelineDefinition.groovy

```diff
@@ -0,0 +1,83 @@
+/*
+ * Main user retirement pipeline
+ */
+pipeline {
+    agent {
+        // Try to run on standard jenkins-worker instances, otherwise the
+        // backup-runner will suffice as fallback.  Master is currently
+        // designated as a backup-runner.
+        label 'jenkins-worker||backup-runner'
```

Comment:
jenkins-worker instances are the default label for this job, but the build jenkins docker container won't work with AWS out of the box.  I noticed that the jenkins master had the "backup-runner" label, so I decided to make it a fallback runner for this pipeline.

Reply:
I understand why you did this, but am concerned that it won't be treated as a fallback but rather as OR by the label picker. In the case that all jenkins-workers are occupied, this would run on the master node. Is this job going to be particularly resource heavy?